qa/L0_batcher/test.sh

```diff
@@ -99,7 +99,7 @@ source ../common/util.sh
 RET=0
```

Comment:
Why the permission of the file has been changed?

Reply:
Not sure how this happened. Will see if it exists after L0_batcher changes are merged in, otherwise, will revert permissions.